tests/_dummies/baseline.py

```diff
@@ -19,7 +19,7 @@
 
 
 # fmt: on
-class DummyClassificationBaseline:
+class DummyClassificationBaseline(LightningModule):
```

Comment:
Why is the inheritance needed here?

Reply:
I added the inheritance both here and for the ResNet class to resolve warnings about functions (like `cli_main`) that required a `LightningModule` as argument, since DummyClassificationBaseline was not considered a `LightningModule`. But maybe there is a better way to do that than by inheritance.

Reply:
Fixed (see below)